helm/cloud-provider-cloud-director-app/charts/cloud-provider-for-cloud-director/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: cloud-provider-for-cloud-director
+version: 0.0.0
+appVersion: 1.1.1
+home: https://github.com/giantswarm/cloud-provider-for-cloud-director
+sources:
+- https://github.com/giantswarm/cloud-provider-for-cloud-director
+description: A helm chart for deploying the CPI of VMware Cloud Director (VCD) infrastructure provider (CAPVCD).
+keywords:
+- capvcd
+- vmware
+- cloud
+- cpi
+- director
+- vcd
+annotations:
+  application.giantswarm.io/team: "rocket"
+maintainers:
+- name: Team Rocket
+  email: team-rocket@giantswarm.io
+restrictions:

```

Comment:
Not sure about this either. The app should only be installed already on the correct installations.